addons/hr_timesheet/views/project_task_views.xml

```diff
@@ -91,32 +91,52 @@
                         </form>
                     </field>
                     <group invisible="not analytic_account_active">
-                        <group class="oe_subtotal_footer" name="project_hours">
-                            <span class="o_td_label float-start">
-                                <label class="fw-bold" for="effective_hours" string="Time Spent"/>
-                            </span>
-                            <field name="effective_hours" widget="timesheet_uom" nolabel="1"/>
-                            <button name="action_view_subtask_timesheet" type="object" class="ps-0 border-0 oe_inline oe_link mb-2 o_td_label float-start" invisible="subtask_effective_hours == 0.0">
-                                <span class="text-nowrap">Time Spent on Sub-tasks:</span>
-                            </button>
-                            <field name="subtask_effective_hours" class="mt-2" widget="timesheet_uom"
-                                   invisible="subtask_effective_hours == 0.0" nolabel="1"/>
-                            <span invisible="subtask_effective_hours == 0.0" class="o_td_label float-start my-1">
-                                <label class="fw-bold" for="total_hours_spent"
-                                       invisible="subtask_effective_hours == 0.0 or encode_uom_in_days"/>
-                                <label class="fw-bold" for="total_hours_spent"
-                                       invisible="subtask_effective_hours == 0.0 or not encode_uom_in_days"/>
-                            </span>
-                            <field name="total_hours_spent" widget="timesheet_uom" class="oe_subtotal_footer_separator" nolabel="1"
-                                   invisible="subtask_effective_hours == 0.0" />
-                            <span class="o_td_label float-start my-1" invisible="allocated_hours == 0.0">
-                                <label class="fw-bold" for="remaining_hours"
-                                       invisible="remaining_hours &lt; 0"/>
-                                <label class="fw-bold text-danger" for="remaining_hours"
-                                       invisible="remaining_hours &gt;= 0"/>
-                            </span>
-                            <field name="remaining_hours" widget="timesheet_uom" class="oe_subtotal_footer_separator"
-                                   invisible="allocated_hours == 0.0" nolabel="1" decoration-danger="remaining_hours &lt; 0"/>
+                        <group class="oe_subtotal_footer d-flex justify-content-end w-100" style="margin-bottom:0 !important;" name="project_hours">
```

Comment:
- `w-100` does not seem useful
- `style` added could be replaced by `mb-0` bootstrap class 🙂 
```suggestion
                        <group class="oe_subtotal_footer d-flex justify-content-end mb-0" name="project_hours">
```